src/main/java/core/basesyntax/Drowable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drowable {
+    void drow();
```

Comment:
```suggestion
public interface Drawable {
    void draw();
```

Reply:
you didnt fix this
![image](https://user-images.githubusercontent.com/81781479/191530173-610830f4-68b6-4175-988f-db0fddd342d5.png)